instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -150,7 +149,7 @@ tasks {
     dependsOn(testing.suites)
   }
 
-  compileTestJava {
+  compileJava {
```

Comment:
why do you need to do this?

Reply:
I got a warning without it

Reply:
an existing warning or a new warning related to this PR?

Reply:
what's the warning?

Reply:
apprently it wasn't needed :wink: